etc/p3-config/p3-config.yml

```diff
@@ -586,7 +569,7 @@ p3_image_fedora:
 
 # Build of OpenHPC on CentOS
 p3_image_openhpc:
-  name: "CentOS7.4-OpenHPC"
+  name: "CentOS7.5-OpenHPC"
```

Comment:
What's this? I think we just merged that change

Reply:
Oh yeah!

Reply:
I've removed them.